polaris/polaris/templates/deposit/form.html

```diff
@@ -16,13 +16,18 @@
 
     {% for field in form %}
     <div class="field">
+      {% if field.name == 'amount' %}
```

Comment:
Is fa-dollar-sign a bulma attribute? If so, can we pass it in in the `django.Form` instead of special casing amount here?

Reply:
I don't think so because the `<span>` tag and its contents are a separate component from the form field. The `fa-dollar-sign` attribute probably needs to live in an `<i>` tag too.